win32/ctags_vs2017.vcxproj

```diff
@@ -0,0 +1,310 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think adding "ctags_vs2017.vcxproj" is needed for "Use sourceformat free/variable".